src/product-type-export.js

```diff
@@ -246,11 +247,22 @@ export default class ProductTypeImport {
     })
   }
 
+  getProductTypeClient () {
+    const clientEndpoint = this.client.productTypes
+
+    if (this.config.where)
+      clientEndpoint.where(this.config.where)
+
+    return clientEndpoint
+  }
+
   downloadProductTypes (file) {
     const writeStream = createWriteStream(file)
     writeStream.write('[')
     let isFirst = true
-    return this.client.productTypes.process(({
+
+    return this.getProductTypeClient()
+    .process(({
```

Comment:
Yeah I guess it's also personal preference. I think indenting those chains makes it a more readable as you can easily see where the chain comes from.